lib/analysis_options.next.yaml

```diff
@@ -0,0 +1,210 @@
+analyzer:
```

Comment:
Can you rename the file to be "7.0.0" instead of "next"? I'm going to try merging #113 in so that the changes are clear in this Pull Request. Currently is quite difficult to evaluate what lints are changing.

Reply:
#113 is now in, updating with main and renaming should improve the reported difference.